src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/ProfessionalExternalUserControllerProviderTest.java

```diff
@@ -193,6 +205,73 @@ private ProfessionalUser setupInteractionsForProfessionalUser() throws JsonProce
         return professionalUser;
     }
 
+    private ProfessionalUser setupInteractionsForProfessionalUserWithUserAccessTypes() throws JsonProcessingException {
```

Comment:
don't see new provider test for adding new user to an organisation

Reply:
I did not change anything about adding a new user on this pr